tests/test_connection.py

```diff
@@ -93,25 +93,24 @@ def test_get_wiql(self, tfsapi):
 
     @pytest.mark.httpretty
     def test_get_projects(self, tfsapi):
```

Comment:
test_get_projects => test_projects?

Can we test both methods for backward compatibility for a while?